src/privileged/vpnRecommender/Doorhanger.jsm

```diff
@@ -79,12 +77,12 @@ class Doorhanger {
     panel.setAttribute("level", "parent");
 
     if (Services.appinfo.OS === "Darwin") {
-	    panel.style.height = "355px";
-	    panel.style.width = "282px";
-	  } else {
-		  panel.style.height = "355px";
-		  panel.style.width = "282px";
-	  }
+      panel.style.height = "355px";
+      panel.style.width = "282px";
+    } else {
+      panel.style.height = "355px";
+      panel.style.width = "282px";
+    }
```

Comment:
Unrelated, but not sure if we need this `if()` since the values seem to be the same in each code branch.